extensions/neo4j/runtime/src/main/java/io/quarkus/neo4j/runtime/Neo4jConfiguration.java

```diff
@@ -57,6 +75,65 @@
         public boolean disabled = false;
     }
 
+    @ConfigGroup
+    static class TrustSettings {
+
+        public enum Strategy {
+
+            TRUST_ALL_CERTIFICATES,
+
+            TRUST_CUSTOM_CA_SIGNED_CERTIFICATES,
+
+            TRUST_SYSTEM_CA_SIGNED_CERTIFICATES
+        }
+
+        /**
+         * Configures which trust strategy to apply when using encrypted traffic.
+         */
+        @ConfigItem(defaultValue = "TRUST_SYSTEM_CA_SIGNED_CERTIFICATES")
+        public Strategy strategy = Strategy.TRUST_SYSTEM_CA_SIGNED_CERTIFICATES;
```

Comment:
```suggestion
        public Strategy strategy;
```

Reply:
Oh I just saw that there is test `defaultsShouldWork()` so maybe let's keep it the way it was.